src/main/java/com/redhat/developers/msa/ola/OlaController.java

```diff
@@ -104,22 +104,35 @@ public String health() {
      * @return The feign pointing to the service URL and with Hystrix fallback.
      */
     private HolaService getNextService() {
-        // This stores the Original/Parent ServerSpan from ZiPkin.
-        final ServerSpan serverSpan = brave.serverSpanThreadBinder().getCurrentServerSpan();
-        final CloseableHttpClient httpclient =
-            HttpClients.custom()
-                .addInterceptorFirst(new BraveHttpRequestInterceptor(brave.clientRequestInterceptor(), new DefaultSpanNameProvider()))
-                .addInterceptorFirst(new BraveHttpResponseInterceptor(brave.clientResponseInterceptor()))
-                .build();
+        String url = System.getenv("HOLA_SERVER_URL");
```

Comment:
Just wondering if this code can be simplified, but just relying on the HOLA_SERVER_URL being provided? Is there a reason for supporting URL or HOST/PORT - i.e. are they needed for different situations?